From 4f5217ad32933aa0dbd6665d908e08db301e7a28 Mon Sep 17 00:00:00 2001
From: Juhyung Park <qkrwngud825@gmail.com>
Date: Tue, 1 Mar 2022 20:52:13 +0900
Subject: [PATCH 3/7] builddeb: do not compress deb files

Signed-off-by: Juhyung Park <qkrwngud825@gmail.com>
---
 scripts/package/builddeb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 91a502bb97e8..6af20a67f17b 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -50,7 +50,7 @@ create_package() {
 
 	# Create the package
 	dpkg-gencontrol -p$pname -P"$pdir"
-	dpkg-deb $dpkg_deb_opts ${KDEB_COMPRESS:+-Z$KDEB_COMPRESS} --build "$pdir" ..
+	dpkg-deb $dpkg_deb_opts -Znone --build "$pdir" ..
 }
 
 deploy_kernel_headers () {
-- 
2.35.1

