From 77a6cdb81edec2781ab638a6861bc5b3b9d6678f Mon Sep 17 00:00:00 2001
From: Juhyung Park <qkrwngud825@gmail.com>
Date: Tue, 1 Mar 2022 20:52:13 +0900
Subject: [PATCH 03/10] builddeb: do not compress deb files

Signed-off-by: Juhyung Park <qkrwngud825@gmail.com>
---
 scripts/package/builddeb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index d7dd0d04c70c9..edc36bf005d5a 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -48,7 +48,7 @@ create_package() {
 
 	# Create the package
 	dpkg-gencontrol -p$pname -P"$pdir"
-	dpkg-deb $dpkg_deb_opts ${KDEB_COMPRESS:+-Z$KDEB_COMPRESS} --build "$pdir" ..
+	dpkg-deb $dpkg_deb_opts -Znone --build "$pdir" ..
 }
 
 install_linux_image () {
-- 
2.43.0

